### PR TITLE
Replace Android example with a modern example, using rtmp-rtsp-stream-client-java

### PR DESCRIPTION
**Pull request by GeneticGenesis**
Our Android example was overdue some love.

This PR replaces the old Javacv based Android example with a new example based on rtmp-rtsp-stream-client-java with a variety of reliability features built in, and clean, easy to understand code.

### CODE REVIEW

File: android-live-streaming/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+* Built in Android Studio 4.1
+* Dependencies are managed using Gradle
+
+# Support
+
+Mux will provides basic support for the features shown in this example. Please raise a GitHub issue if you run into problems, but remember that we are limited to functionality that rtmp-rtsp-stream-client-java provides.
+
+# Documentation
+
+This project uses [rtmp-rtsp-stream-client-java RTMP library](https://github.com/pedroSG94/rtmp-rtsp-stream-client-java) to reliably broadcast a live stream from an Android device to an RTMP endooint, in this case, [Mux](https://mux.com/).
```
Comment (jsanford8) [inline android-live-streaming/README.md:51]: endooint -> endpoint

File: android-live-streaming/README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+This Activity acts as the entrypoint of the application, and allows the user to enter a Stream Key.
+
+### BroadcastActivity.java
+
+Most of the application is implemented in BroadcastActivity.swift, it's fairly short, around 250 lines right now, and is deliberately not broken into multiple files or classes. Here's an overview of how it works. 
+
+#### onCreate()
+
+When the application is started, the main entrypoint, `onCreate()` is called. First, this then performs basic boilerplate to setup the UI elements, including the `surfaceView`, which is used as the preview for the camera.
```
Comment (jsanford8) [inline android-live-streaming/README.md:69]: "First, this then performs" -> "First, this performs"

File: android-live-streaming/README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+When the application is started, the main entrypoint, `onCreate()` is called. First, this then performs basic boilerplate to setup the UI elements, including the `surfaceView`, which is used as the preview for the camera. 
+
+```java
+    // Init the Surface View for the camera preview
+    surfaceView = findViewById(R.id.surfaceView);
+    surfaceView.getHolder().addCallback(this);
+```
+
+Next is instantiates the `RTMPCamera1`, passing in a reference to the preview, and setting up a callback for the FPS change events.
```
Comment (jsanford8) [inline android-live-streaming/README.md:77]: "Next, is instantiates" -> "Next, it instantiates"

File: android-live-streaming/app/proguard-rules.pro [hunk 4]
```diff
@@ -18,4 +18,4 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
+#-renamesourcefileattribute SourceFile
```
Comment (jsanford8) [inline android-live-streaming/app/proguard-rules.pro:21]: need newline? Or is this just how android studio does things?

File: android-live-streaming/app/src/main/java/com/example/muxlive/BroadcastActivity.java [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+    @Override
+    public void onAuthSuccessRtmp() {
+    }
+
+    // Touch Listener Callbacks
+    @Override
+    public boolean onTouch(View view, MotionEvent motionEvent) {
+        return false;
+    }
+}
```
Comment (jsanford8) [inline android-live-streaming/app/src/main/java/com/example/muxlive/BroadcastActivity.java:287]: Likewise here, maybe an android studio setting?

File: android-live-streaming/app/src/test/java/com/example/muxlive/ExampleUnitTest.java [hunk 6]
```diff
@@ -11,7 +11,7 @@
  */
 public class ExampleUnitTest {
     @Test
-    public void addition_isCorrect() throws Exception {
+    public void addition_isCorrect() {
```
Comment (jsanford8) [inline android-live-streaming/app/src/test/java/com/example/muxlive/ExampleUnitTest.java:14]: I love these template tests

File: android-live-streaming/gradle/wrapper/gradle-wrapper.properties [hunk 7]
```diff
@@ -1,6 +1,6 @@
-#Thu Jan 03 16:27:49 PST 2019
+#Thu Dec 03 11:36:30 GMT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (jsanford8) [inline android-live-streaming/gradle/wrapper/gradle-wrapper.properties:6]: ooh you on the new new

**Review by jsanford8 — APPROVED** [review]
Looks good to me, just a few doc cleanup typos and maybe an android studio setting change (though that's not required at all)

File: android-live-streaming/gradle/wrapper/gradle-wrapper.properties (hunk 7 above)
Comment (GeneticGenesis) [reply android-live-streaming/gradle/wrapper/gradle-wrapper.properties:6]: 💅

File: android-live-streaming/app/src/test/java/com/example/muxlive/ExampleUnitTest.java (hunk 6 above)
Comment (GeneticGenesis) [reply android-live-streaming/app/src/test/java/com/example/muxlive/ExampleUnitTest.java]: Killed em

**Pull request merged**